packages/mui-material-next/src/LinearProgress/LinearProgress.types.ts

```diff
@@ -18,9 +18,15 @@ export interface LinearProgressOwnProps {
    * @default 'primary'
    */
   color?: OverridableStringUnion<
-    'primary' | 'secondary' | 'error' | 'info' | 'success' | 'warning' | 'inherit',
+    'primary' | 'secondary' | 'tertiary',
     LinearProgressPropsColorOverrides
   >;
+  /**
+   * If `true`, the component render indeterminate or query mode using four colors instead of one.
+   * This only works if variant is `indeterminate` or `query`.
+   * @default false
+   */
+  fourColor?: boolean;
```

Comment:
I could find the four colors in the spec, but I couldn't find any example of how it should look. Do you know if there are some?